blockend/public-kernel.js

```diff
@@ -66,6 +68,33 @@ export default class PublicKernel extends SimpleKernel {
   gc (at) {
     return this.store.gc(at)
   }
+
+  /**
+   * Bump function
+   */
+  async bump (rantId) {
+    if (!Buffer.isBuffer(rantId)) rantId = Buffer.from(rantId, 'hex')
+    const branch = await this.repo.resolveFeed(rantId)
+    const data = {
+      emo: '💩'
+    }
+    await this.createBlock(branch, TYPE_BUMP, data)
+
+    // branch.inspect() <-- debugging purposes
+  }
+
+  /**
+   * This is used to get the clients key before any actions have been made.
+   * There is probably a better and less hacky way to do it.
+   */
```

Comment:
+points for hacky solution. there's `this.pk` would that work?
https://github.com/telamon/picostack/blob/master/simple-kernel.js#L39

Reply:
thx was looking for this x)